buildSrc/src/test/groovy/org/elasticsearch/gradle/doc/RestTestsFromSnippetsTaskTest.groovy

```diff
@@ -19,32 +19,46 @@
 
 package org.elasticsearch.gradle.doc
 
-import org.elasticsearch.gradle.doc.SnippetsTask.Snippet
+import org.elasticsearch.gradle.GradleUnitTestCase
 import org.gradle.api.InvalidUserDataException
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.ExpectedException
+
+import static org.junit.Assert.*
```

Comment:
Please do not use wildcard imports. We disallowed them in the rest of the codebase.

Reply:
I fixed my IDE setting some time ago, this must pre-date that. Fixed now.